packages/@aws-cdk/aws-chatbot/test/slack-channel-configuration.test.ts

```diff
@@ -118,6 +118,16 @@ describe('SlackChannelConfiguration', () => {
     });
   });
 
+  test('allows adding a Topic to the existing SlackChannel', () => {
+    const slackChannelFromArn = chatbot.SlackChannelConfiguration.fromSlackChannelConfigurationArn(stack, 'RetrievedSlackChannel', 'arn:aws:chatbot::123456789012:chat-configuration/slack-channel/my-slack');
+
+    const topic = new sns.Topic(stack, 'MyTopic');
+
+    slackChannelFromArn.addNotificationTopic(topic);
+
+    Template.fromStack(stack).resourceCountIs('AWS::SNS::Topic', 1);
```

Comment:
I may be misunderstanding how this works, but this test case only tests that a `Topic` resource is created. It does not check that there is an association between the topic and the imported chatbot configuration.

Reply:
That is true 👍 I've tested this now and the in the template that's created, there's no reference to [SnsTopicArns](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-chatbot-slackchannelconfiguration.html#cfn-chatbot-slackchannelconfiguration-snstopicarns).